app/mailers/application_mailer.rb

```diff
@@ -1,3 +1,4 @@
+# App Mailer
```

Comment:
You can delete things like the Mailer if you don't need it. It is good practice to clean up after the project creation and not have files in the project, that are not needed.